lib/core-integration/src/Test/Integration/Scenario/API/Byron/Transactions.hs

```diff
@@ -98,27 +99,27 @@ spec = do
             , expectField (#balance . #available) (`shouldBe` Quantity faucetAmt)
             ]
 
-    it "BYRON_RESTORE_09 - Ledger wallet" $ \ctx -> do
-        -- NOTE
-        -- Special legacy wallets where addresses have been generated from a
-        -- seed derived using the auxiliary method used by Ledger.
-        let mnemonics =
```

Comment:
I don't know how to generate this special legacy ledger wallet. For the moment I commented out this test.